Source/Engine/Scripting/Internal/EngineInternalCalls.cpp

```diff
@@ -117,7 +118,7 @@ DEFINE_INTERNAL_CALL(void) ProfilerInternal_BeginEvent(MString* nameObj)
 #if COMPILE_WITH_PROFILER
     StringView name;
     MUtils::ToString(nameObj, name);
-    ProfilerCPU::BeginEvent(*name);
+    ManagedEventsCPU.Push(ProfilerCPU::BeginEvent(*name));
```

Comment:
Oh, what's this?

Reply:
small fix for c# side what is removed and added to the Profiler stack the GPU profiler has this so i added it to CPU side